setup.py

```diff
@@ -27,20 +27,10 @@
     python_requires=">=3.6.*",
     install_requires=[
         'clickhouse-driver~=0.1.2',
```

Comment:
```suggestion
        'clickhouse-driver~=0.1.3',
```

Since you are using DB API, we have to upgrade driver to the first version that supported it: https://github.com/mymarilyn/clickhouse-driver/blob/master/CHANGELOG.md#013---2020-02-21

Reply:
Updated